src/test/kotlin/org/rust/lang/core/resolve/RsPackageLibraryResolveTest.kt

```diff
@@ -602,6 +602,14 @@ class RsPackageLibraryResolveTest : RsResolveTestBase() {
         } //^ unresolved
     """)
 
+    fun `test macro 2 (unresolved when absolute path)`() = stubOnlyResolve("""
```

Comment:
Hmm, is it a test for the (default) 2021 edition?

Reply:
Yes, it checks that macro is unresolved with 2021 edition

Reply:
But shouldn't we add a test for 2015 edition?

Reply:
Added